public/vendor/flot/jquery.flot.gauge.js

```diff
@@ -529,6 +584,11 @@
                     drawThresholdValue(gaugeOptionsi, layout, cellLayout, i + "_" + j, threshold.value, a);
                 }
             }
+            
+            var neutral = gaugeOptionsi.gauge.neutralValue;
+            if (neutral != null) {
```

Comment:
This needs a check for if neutral > min, to avoid things like this (notice the -10 and -11 overlapping on the left):

<img width="1275" alt="Screenshot 2022-11-16 at 16 49 11" src="https://user-images.githubusercontent.com/100691367/202242518-ce19c70f-9381-41a6-a8d4-fdfb4eb9abbe.png">

Reply:
&& neutral < max